lib/KCard/index.vue

```diff
@@ -320,7 +320,7 @@
             },
             thumbnailStyles: {
               ...thumbnailCommonStyles,
-              borderRadius: '8px 0 0 8px',
+              borderRadius: '0 8px 8px 0',
```

Comment:
Hi @Spoorthy1423 . Thanks for you work here. I have reviewed it and I think it's still breaking when it's align left.
<img width="1355" alt="Screenshot 2024-08-26 at 16 05 16" src="https://github.com/user-attachments/assets/736080cf-661c-4868-8144-0fb4795139be">

Reply:
I guess we will need to swap it  when the thumbnail is on the right
`borderRadius: '0 8px 8px 0',`

And when the thumbnail is on the left
`borderRadius: '8px 0 0 8px',`

Reply:
Once we confirm that everything is looking good we will go a head and merge it.